src/number_theory/number_field/embeddings.lean

```diff
@@ -371,6 +402,7 @@ lemma mk_complex.filter (w : { w : infinite_place K // w.is_complex }) :
         complex_embedding.is_real_conjugate_iff.not.2 (is_complex_iff.1 w.2)⟩ } :=
 begin
   ext φ,
+  have := mk_complex_embedding K φ,
```

Comment:
this shouldn't be needed, right?

Reply:
Yes, you're right. I forgot to clean up. Thanks!